src/components/Chart/Table.js

```diff
@@ -0,0 +1,228 @@
+            }
+      )
+    } else {
+      setSortBy({ key, order: 'desc' })
+    }
+  }
+
+  let currentColorRange = colorRanges[colorRange] || colorRange
+  let colorScale
+  let domain
+
+  if (thresholds) {
+    domain = thresholds
+    if (!colorRange) {
+      currentColorRange = colorRanges.sequential.slice(0, domain.length + 1)
+    }
+    colorScale = scaleThreshold()
+      .domain(domain)
+      .range(currentColorRange || colorRanges.sequential)
+  } else {
+    const colorColumns = tableColumns.filter(c => c.color).map(c => c.column)
+    const colorValues = !colorColumns.length
+      ? []
+      : parsedData
+          .reduce((values, row) => {
+            colorColumns.forEach(key => {
+              values.push(row[key])
+            })
+            return values
+          }, [])
+          .filter(Boolean)
+          .filter(deduplicate)
+
+    colorScale = getColorMapper(props, colorValues)
+  }
+
+  return (
+    <div {...styles.container}>
+      <table {...styles.table}>
+        <thead>
+          <tr>
+            {columns.map((tableHead, index) => (
+              <th
+                {...styles.header}
+                {...colorScheme.set('borderBottomColor', 'text')}
+                style={{
+                  textAlign: numberColumns.includes(tableHead)
+                    ? 'right'
+                    : 'left',
+                  cursor: 'pointer',
+                  whiteSpace: sortBy.key === tableHead ? 'nowrap' : undefined
+                }}
+                key={index}
+                onClick={() => setSort(columns[index])}
+              >
+                {tableHead}
+                {sortBy.key &&
+                  sortBy.key === tableHead &&
+                  (sortBy.order === 'desc' ? (
+                    <ExpandMoreIcon style={{ paddingLeft: '2px' }} />
+                  ) : (
+                    <ExpandLessIcon style={{ paddingLeft: '2px' }} />
+                  ))}
+              </th>
+            ))}
+          </tr>
+        </thead>
+        <tbody style={{ marginTop: '5px' }}>
+          {parsedData.map((row, rowIndex) => (
+            <tr
+              key={rowIndex}
+              {...(rowIndex % 2 !== 0 &&
+                colorScheme.set('backgroundColor', 'hover'))}
+            >
+              {columns.map((cellKey, cellIndex) => (
+                <Cell
+                  key={cellIndex}
+                  type={tableColumns.find(d => d.column === cellKey)?.type}
+                  width={tableColumns.find(d => d.column === cellKey)?.width}
+                  color={tableColumns.find(d => d.column === cellKey)?.color}
+                  value={row[cellKey]}
+                  colorScale={colorScale}
+                >
+                  {tableColumns.find(d => d.column === cellKey)?.type ===
+                  'number'
+                    ? getFormat(numberFormat)(row[cellKey])
+                    : row[cellKey]}
```

Comment:
```suggestion
                  {numberColumns.includes(tableHead)
                    ? getFormat(numberFormat)(row[cellKey])
                    : row[cellKey]}
```

and move the `getFormat(numberFormat)` outside of render of individual cells as it only needs to be done once